vue-frontend/src/components/ios-app-development/BlogSection.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+  <section class="tw-my-20 md:tw-my-32 xll:tw-container">
+    <div
+      class="tw-pb-4 md:tw-py-12 tw-text-center tw-font-inter-bold tw-text-3xl tw-leading-[2.4375rem] md:tw-text-[2.65625rem] md:tw-leading-[3.796875rem] lg:tw-text-[3.4375rem] lg:tw-leading-[5.15625rem] tw-tracking-[-4%] xll:tw-py-24 tw-text-black-core/[0.87]"
+    >
+      <span>Our Blogs</span>
+    </div>
+    <!-- Mobile UI start -->
+    <div class="tw-mt-8 tw-block md:tw-hidden">
+      <swiper
+        :grabCursor="true"
+        :effect="'creative'"
+        :centeredSlides="true"
+        :loop="false"
+        :creativeEffect="{
+          limitProgress: 5,
+          loop: false,
+          next: {
+            translate: [800, 0, 0],
+          },
+          prev: {
+            translate: [0, 0, -100],
+          },
+        }"
+        :breakpoints="{
+          '450': {
+            creativeEffect: {
+              limitProgress: 5,
+              next: {
+                translate: [450, 0, 0],
+              },
+              prev: {
+                translate: [0, 0, -100],
+              },
+            },
+          },
+        }"
+        :slidesPerView="1.7"
+        :spaceBetween="10"
+        :modules="modules"
+      >
+        <swiper-slide
+          v-for="(blog, index) in blogs"
+          :key="index"
+          class="tw-mt-8"
+        >
+          <div
+            :class="blog.className"
+            class="tw-w-60 xs:tw-w-80 tw-h-[350px] tw-drop-shadow-md tw-text-left tw-cursor-pointer"
```

Comment:
increase some card width in below of 420 screen width
![Screenshot from 2023-09-22 17-19-23](https://github.com/canopas/canopas-website/assets/115449373/9297446c-6797-4d7b-923d-7a2485271b67)

Reply:
its cutting in below xs devices.tha's why i reduced . in real device iphone it was cutting.